stable/docker-registry/templates/ingress.yaml

```diff
@@ -2,7 +2,11 @@
 {{- $serviceName := include "docker-registry.fullname" . -}}
 {{- $servicePort := .Values.service.port -}}
 {{- $path := .Values.ingress.path -}}
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
can maybe just do like in https://github.com/helm/charts/pull/22319/files#diff-801a58d4d0e066588b81b5f17849c124R2 ?

Reply:
@davidkarlsen I've changed the condition. When I try to use:
```
apiVersion: {{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" -}} networking.k8s.io/v1beta1 {{- else }} extensions/v1beta1 {{- end }}
```
it fails on Go version 1.13.10

Reply:
See updated PR with less chomping.

Reply:
@davidkarlsen thank you, I did it like in the PR